coverage_config_aarch64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 76.2,
+  "coverage_score": 76.7,
```

Comment:
Hmm, why is this changed? Shouldn't it be changed upstream as well?

Reply:
I don't know, could it be because of different kvm-bindings?

You just changed it upstream and no other changes went in in the meantime.